pages/Network_APIs/WorkflowforContribution.md

```diff
@@ -0,0 +1,69 @@
+  * The production crew have a set of credentials (SIM/eSIM) of the network devices will connect to. The network provides, by default, best-effort connectivity.
+ 
+## Before the Event: 
+ 
+### Step 1) Preparing your device, configuring application clients & servers and configuring client/server flows
+  * Devices have already best effort connectivity .
+  * Through app specific [AppAPI], the client app is properly connected to the Application Server. e.g. Haivision app with BBC server, or LiveU app with LiveU server, etc …
+ 
+### Step 2)Event planning and pre-booking
+  * a) Through the Network API Platform, the production crew (on location or located in production centre) can discover the capabilities the network can offer in a particular location and at a particular time (for which the production company is eligible for).
+    * Example: QoD available, connectivity monitoring available, Timing as a service available, edge compute instantiation, …
+  * b) Through the Network API Platform, the production crew requests network services for every of the planned SIM cards in advance. Possible services (Network Capabilities) are:
+    * Quality on Demand
+      * One or several QoS profiles for each SIM card (QOS Profiles are mapped to 5QIs) 
+      * Example: A sim may be pre booked for One uplink video / One uplink audio / one downlink data / …
+    * Time as a service
+      * Access stratum based or PTP based
+    * Note: Booking is done based on :
+      * Geographical area
+      * Schedule (starting time and closing time of the event)
+  * c) Through the Network API Platform, the production receives a booking offer responding to the service request.
+  * d) Through the Network API Platform the production accepts the service booking offer (involving payment/contract/SLA aspects)[TL6] .
+  * e) Through the production receives connection IDs to be used for every reserved SIM/services
+ 
+## During the Event: 
+ 
+### Step 3) Local configuration
+  * a) Production crew arrives in the venue, plugs the SIM cards and turn on the devices, connectivity is enabled based on the booked network services (See step 2).
+  * b) The production crew uses [AppAPI] to configure the different services, matching the connection ID delivered in step 2-e).
+    * Example: QoD service: A camera for which  one video + one audio is pre-booked. The [AppAPI] is used to properly configure the bitrate of the audio and video output, and the provided IDs.
+    * Example: Time Sync service: A camera for which access to global clock is requested. The [AppAPI] is used to properly configure the time parameters and the provided IDs.
+ 
+Independent steps that can be triggered during the life of the event:
+  * The production crew can use [GwAPI] to monitor that the flows are coming and are properly using the reserved resource.
+  * The production crew receives notification through [GwAPI] indicating potential issues (throughput, delay, …)
```

Comment:
[GwAPI] Network API Platform

Reply:
all occurrences of [GwAPI] should be replace with Network API Platform